classes/cluster/mk24-qa-bare-metal-01/openstack/compute.yml

```diff
@@ -0,0 +1,53 @@
+classes:
+- system.linux.system.repo.mos9
+- system.linux.system.repo.contrail
+- system.nova.compute.cluster
+- service.ceilometer.agent.cluster
+- service.ceilometer.agent.publisher.graphite
```

Comment:
I believe for dvr we should  include system.linux.network.interface.ovs_lab_dvr for compute as well